src/icons/EyeIcon.tsx

```diff
@@ -21,7 +21,7 @@ function EyeIcon({ color, size }: Props) {
 
 EyeIcon.defaultProps = {
   size: 24,
-  color: Colors.black,
+  color: Colors.textPrimary,
```

Comment:
Unrelated to this PR I wonder if we should have `iconPirmary` and `iconInverse`.